src/Parser/ParseDate.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+
+namespace Silarhi\Cfonb\Parser;
+
+
+use Silarhi\Cfonb\Exceptions\ParseException;
+
+class ParseDate
```

Comment:
Could you please rename `ParseDate` class to `DateParser`?

Reply:
done